src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people
+    .filter(
+      human =>
+        century
+          ? (human.sex === 'm' && Math.ceil(human.died / 100) === century)
+          : human.sex === 'm'
+    );
```

Comment:
```suggestion
  const men = people.filter(human =>
        century
          ? (human.sex === 'm' && Math.ceil(human.died / 100) === century)
          : human.sex === 'm'
    );
```

Not necessary to start `.filter` from a new line, the same for `human`,  it  would be better to keep it on one line for readability